chia/cmds/wallet_funcs.py

```diff
@@ -230,3 +230,19 @@ async def execute_with_wallet(wallet_rpc_port: int, fingerprint: int, extra_para
             print(f"Exception from 'wallet' {e}")
     wallet_client.close()
     await wallet_client.await_closed()
+
+
+async def create_new_wallet(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
+    data = await wallet_client.create_new_wallet(args["wallet_type"], args["data"])
+    if data and data["success"]:
+        if args["wallet_type"] == "cc_wallet" and args["data"]["mode"] == "new":
+            print("New colour created: ", data["colour"])
+        elif args["wallet_type"] == "cc_wallet" and args["data"]["mode"] == "existing":
```

Comment:
This if is repeated, structuring can factor it out

Reply:
chia/cmds/wallet_funcs.py

Reply:
Done